### PR TITLE
timescaledb: add postgresql extension timescaledb 2.16.0

### PR DESCRIPTION
**Pull request by cgrzemba**
This extension is a prerequisite for Zabbix server 7

### CODE REVIEW

File: components/database/postgresql-16-timescaledb/Makefile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+COMPONENT_ARCHIVE=	$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:c68c12c3d62f2e3c46d277d2558c20e31d3826b84e15a8594d1084874a1ea9a4
+COMPONENT_ARCHIVE_URL=	https://github.com/timescale/timescaledb/archive/refs/tags/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	www.timescale.com
+COMPONENT_FMRI=	database/postgres-$(PG_VERNUM)/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	System/Databases
+COMPONENT_LICENSE=	Apache-2.0 TSL
+COMPONENT_LICENSE_FILE=	LICENSE
+
+include $(WS_MAKE_RULES)/common.mk
```
Comment (AndWac) [inline components/database/postgresql-16-timescaledb/Makefile:32]: TEST_TARGET= $(NO_TESTS)
is missing as gmake test fails with an error message.

File: components/database/postgresql-16-timescaledb/Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+COMPONENT_BUILD_ACTION= cd $(@D); $(ENV) $(COMPONENT_BUILD_ENV) ./bootstrap -DUSE_OPENSSL=0 && cd build && gmake
+
```
Comment (AndWac) [inline components/database/postgresql-16-timescaledb/Makefile:35]: OpenSSL is explicitly disabled here. Is there a reason for this?

Comment (cgrzemba) [reply components/database/postgresql-16-timescaledb/Makefile:35]: yes, bootstrapping fails with

CMake Error at CMakeLists.txt:593 (message):
  PostgreSQL was built without OpenSSL support, which TimescaleDB needs for
  full compatibility.  Please rebuild PostgreSQL using `--with-openssl` or if
  you want to continue without OpenSSL, re-run bootstrap with
  `-DUSE_OPENSSL=0`
but it is not true and the extension works for me.

Comment (AndWac) [reply components/database/postgresql-16-timescaledb/Makefile:35]: Ok, you should probably add a comment with that information then.

**Pull request merged**